packages/saber-plugin-image/lib/saber-browser.js

```diff
@@ -13,16 +13,22 @@ export default ({ Vue }) => {
 
       if ($el.dataset.src || $el.dataset.srcset) {
         lozad($el, {
-          loaded(el) {
-            el.addEventListener(
-              'load',
-              () => el.setAttribute('data-lazy-loaded', ''),
-              { once: true }
-            )
+          loaded: el => {
+            el.addEventListener('load', () => {
+              el.setAttribute('data-lazy-loaded', '')
+              this.removeBlendIn = setTimeout(() => {
```

Comment:
Ah, I didn't look at your code. Guess listening to `transitionend` is better?

Reply:
Yeah sure, I just went with the easiest way.